CHANGELOG.md

```diff
@@ -4,6 +4,29 @@
 
 ### Improvements
 
+- Add error type identifier to improve obfuscated Flutter issue titles ([#2170](https://github.com/getsentry/sentry-dart/pull/2170))
+  - Example: transforms issue titles from `GA` to `FlutterError` or `minified:nE` to `FlutterError`
+  - This is enabled automatically and will change grouping if you already have issues with obfuscated titles
+  - You can add your custom exception identifier if there are exceptions that we do not identify out of the box
+  - If you want to disable this feature, set `enableExceptionTypeIdentification` to `false` in your Sentry options
```

Comment:
it's great that we have this option 👍 I would put this note one bullet above, so that it's connected to the callout about being opt out and that it changes (fixes!) grouping